policies/rudderc/docs/src/syntax.md

```diff
@@ -0,0 +1,100 @@
+# Technique syntax
+
+## General organization
+
+A technique is made of:
+
+* General metadata: its name, version, documentation, etc.
+* Parameters it can take
+* Resource file that can be attached
+* A method tree (made of block and leaf methods)
+
+## Technique
+
+* `id`: Unique identifier of the technique. Must only contain alphanumeric or underscore characters.
+* `name`: Human-readable name of the technique
+* `version`: Version in the X.Y format.
+* `description` (optional): Single line description of what the technique does.
+* `documentation` (optional): Documentation in Markdown format.
+* `tags` (optional): Optional key-value tags.
+* `category` (optional): Rudder category to put the technique in.
```

Comment:
We need to add this in the jsonschema

Reply:
:+1: indeed

I think this can be done in another PR.